### ISSUE TITLE
zulu mounting in 2 places ?

### ISSUE THREAD

**Issue by JackDinn** [mhogomchungu/zuluCrypt#84]
Iv changed from neon user to neon dev recently and iv moved over my zulu encrypted image but when i mount it via the GUI it gets mounted twice ?
![screenshot_20171227_223311](https://user-images.githubusercontent.com/5557560/34395529-d45947c6-eb58-11e7-96b9-5f3948b8cf85.png)

Im not sure if its to do with zulu or this OS but i would like to get rid of the second mounting if possible, Thx.

**Comment by mhogomchungu** [comment]
What version of zuluCrypt are you using and where did you get it from?

This is a very old problem and i think its time to file a bug report on udisks and ask it to ignore zuluCrypt's mapper path.

I think what is going on here is that 

1. zuluCrypt unlocks your volume and creates an encryption mapper that looks like "/dev/mapper/zuluCrypt-blablabla".
2. zuluCrypt mounts the encryption mapper at "/run/media/private/blablabla".
3. KDE's device notifier(it sits on the tray area) or another tool sees the encryption mapper and asks udisks to mount it.
4. Udisks mounts the encryption mapper for the second time resulting in the two entries you see.

zuluCrypt should be able to unmount both mount points and successfully close the encryption mapper but it can not prevent another tool from mounting its encryption mapper.

At the moment, possible work around is to disable device notifier and see if the problem will go away. If it does then look at its configurations to see if you can disable its auto mount feature or exclude zuluCrypt encryption mappers from its list of volumes it can auto mount.

zuluCrypt ships with zuluMount and you can use it to mount your encrypted volumes and well as non encrypted volumes and hence should be a 100% replacement for device manager.

**Comment by JackDinn** [comment]
It is (Version  : 5.4.0) . I got it from your site https://mhogomchungu.github.io/zuluCrypt/

You are correct, it is the "auto mount" device notifier, I have changed the settings to "not auto mount removable storage" and zulu is mounting it only once now (in the correct path). This is fine as i don't need removable media to be auto mounted anyhow. I think it looks like i might be able to set it up to ignore zulu mounts individually but i'm happy with leaving it switched off.

![screenshot_20171228_072654](https://user-images.githubusercontent.com/5557560/34403511-87c5484a-eba0-11e7-8986-40ec03f0f09a.png)


Many thx for your help.
